src/snowflake/snowpark/modin/plugin/_internal/frame.py

```diff
@@ -102,8 +110,10 @@ def create(
         data_column_pandas_labels: list[Hashable],
         data_column_pandas_index_names: list[Hashable],
         data_column_snowflake_quoted_identifiers: list[str],
+        data_column_types: Optional[list[Optional[SnowparkPandasType]]] = None,
```

Comment:
Should default arguments be at the end since the index_column_... following arguments require values?